pkg/systemd/init.js

```diff
@@ -1269,16 +1269,18 @@ $(function() {
         var unit = "[Unit]\nDescription=";
         var service = "\n[Service]\nExecStart=";
         var timer = "\n[Timer]\n";
-        var service_file = unit + timer_unit.Description + service + timer_unit.Command + "\n[Install]\nWantedBy=default.target";
+        var install = "[Install]\nWantedBy=default.target\n";
```

Comment:
`default.target` is really intended for the user systemd service, not the system one. It might not even work there. It also does the wrong thing when booting into rescue mode. Can you use `multi-user.target` instead, for robustness?